src/test/resources/json/after-step-fails.json

```diff
@@ -0,0 +1,38 @@
+[
+    {
+        "id":"simpleId",
+        "name":"Simple feature",
+        "keyword":"Feature",
+        "elements":[
+            {
+                "keyword":"Scenario",
```

Comment:
I decided to have one simple json file but then I found that some border cases are hard to test. That's a place for further improvements.